backend/database/README.md

```diff
@@ -18,9 +18,14 @@
 6. [Test your migration](#test-a-migration)
 7. Check that [orm.py](../layers/persistence/orm.py) matches up with your changes.
 8. Once you've completed the changes, create a PR to get the functions reviewed.
-9. Once the PR is merged, migrations will be run as part of the deployment process to each env.
-10. [Connect to Remote RDS](#connect-to-remote-rds) to single-cell-dev
-11. In a new terminal, complete the migration in the single-cell-dev test env by running:
+9. Rdev compatibility:
+   1. If your migration requires changes to the seed data file for rdev, make those changes in a **new** seed data file with your migration hash in the name.
+      ```
+      s3://env-rdev-dataportal/database/seed_data_##_a1b2c3d4e5f6.sql
```

Comment:
Whenever we do a migrations doesn't the seed file need to updated? Why don't we make this part of the automated migration process for dev.

Reply:
I suppose automating this could be a future fix.